### PR TITLE
feat(deps): bump swagger-ui to 5.7.2

### PR DESCRIPTION
**Pull request by nateslo**
# Summary

Bumps swagger-ui to [5.7.2](https://github.com/swagger-api/swagger-ui/releases/tag/v5.7.2) for multiple callbacks bug fix

### CODE REVIEW

File: packages/portal/swagger-ui-web-component/package.json [hunk 1]
```diff
@@ -17,7 +17,7 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "swagger-client": "^3.26.8",
-    "swagger-ui": "^5.1.3"
+    "swagger-ui": "~5.7.2"
```
Comment (adamdehaven) [inline packages/portal/swagger-ui-web-component/package.json:20]: Any reason you're using `~` instead of `^`?

Comment (nateslo) [reply packages/portal/swagger-ui-web-component/package.json]: switched back to `^`

**Comment by adamdehaven** [comment]
@nateslo FYI in order to trigger a release, you'll need to add/update a commit to the PR with `feat` instead of `chore` as rhetoric commit message type (conventional commits)

**Review by davidma415 — APPROVED** [review]

**Review by mptap — APPROVED** [review]
Lgtm!

**Review by adamdehaven — APPROVED** [review]

**Pull request merged**